sky/execution.py

```diff
@@ -207,7 +207,10 @@ def _execute(
                             f'{colorama.Style.RESET_ALL}')
                 idle_minutes_to_autostop = 1
             stages.remove(Stage.DOWN)
-            if not down:
+            if down:
+                requested_features.add(
+                    clouds.CloudImplementationFeatures.AUTODOWN)
+            else:
                 requested_features.add(clouds.CloudImplementationFeatures.STOP)
```

Comment:
We should only check these when `idle_minutes_to_autostop >= 0` as `<0` means cancelling the autostop which should be fine to ignore.